pyproject.toml

```diff
@@ -12,7 +12,7 @@ requires-python = ">=3.10"
 dependencies = [
     "numpy>=2.2.5,<2.3.0",
     "scipy==1.15.3",
-    "reachy_mini_motor_controller>=1.2.0",    
+    "reachy_mini_motor_controller>=1.3.0",    
```

Comment:
Is the 1.3.0 available? uv complains that it is not

Reply:
https://github.com/pollen-robotics/reachy-mini-motor-controller/pull/22 needs to be merged first